modules/dxp/apps/analytics/analytics-reports-web/src/main/resources/META-INF/resources/js/components/TotalCount.js

```diff
@@ -50,7 +50,9 @@ function TotalCount({
 				.catch(() => {
 					if (isMounted()) {
 						setValue('-');
-						addWarning();
+						if (!warning) {
+							addWarning();
+						}
```

Comment:
This `useWarning` hook is a bit weird; seems I can _add_ a warning, but there is no way to ever remove it? So I guess I don't know why we need the `if` here; if you add a warning and there already is one, an action will be dispatched, harmlessly, I guess, and `warning` will continue to be `true` in the store.

Reply:
Mmmm...you're right. There is no way to remove it. I'm going to remove those `if`. Thanks.